docs/content/en/integrations/parsers/file/checkmarx_cxflow_sast.md

```diff
@@ -0,0 +1,8 @@
+---
+title: "Checkmarx CxFlow SAST"
+toc_hide: true
+---
+- `Checkmarx CxFlow SAST`: JSON report from Checkmarx Cxflow.
```

Comment:
@biennd279 :
It would be nice, if your description is more mature. You already have a nice description in your PR and I guess it would help others as well:

_CxFlow is a Spring Boot application that can run anywhere Java is installed. CxFlow glues together Checkmarx CxSAST and CxSCA scans with feedback to issue tracking systems via webhooks triggered by SCM events._

Reply:
Yes, I will do it.